docs/specs/component.md

```diff
@@ -167,6 +167,9 @@ sending it, like the `prometheus_exporter` sink, SHOULD NOT publish this metric.
 * Properties
   * `count` - The count of Vector events.
   * `byte_size` - The cumulative in-memory byte size of all events sent.
+  * `output` - For components with multiple outputs, the name of the output that
+    events were sent to. For example, the `remap` transform has an additional
+    `dropped` output.
```

Comment:
I feel like it it'd be nice to standardize on output names. Might make it easier/more discoverable for users if they don't have to look for the unique names

Reply:
True, that'd be great for outputs like `dropped` that multiple transforms might have. That said, I imagine there'll also be user configured outputs (e.g. when route is refactored to use the named outputs functionality) which can't be standardized. Perhaps a section standardizing common outputs can be added to `component.md` once multiple outputs are more heavily used?

Reply:
Sounds like a plan 😄